icons/authentik.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Authentik</title><path d="M13.96 9.069h-.84V7.584h-1.234v3.587H5.721c.34.507.54.961.54 1.129 0 .45-1.446 2.995-3.198 2.995C.8 15.354-.745 12.783.372 10.876c.262-.475.65-.872 1.124-1.149a3.112 3.112 0 0 1 1.567-.423c.977 0 1.86.793 2.448 1.572l.109.148V6.691a3.561 3.561 0 0 1 .176-1.115l.025-.076a1.39 1.39 0 0 1 .029-.076c.011-.029.02-.057.032-.084l.035-.083c.094-.218.209-.426.343-.623l.004-.005a4.619 4.619 0 0 1 .199-.262 4.826 4.826 0 0 1 .227-.249l.004-.004a3.816 3.816 0 0 1 .635-.512l.01-.006a3.46 3.46 0 0 1 .555-.293 3.719 3.719 0 0 1 .702-.22 3.88 3.88 0 0 1 .51-.073l.075-.004.036-.002A3.46 3.46 0 0 1 9.39 3h10.84c.058 0 .116.001.172.004l.037.002.075.004a3.873 3.873 0 0 1 1.212.294l.099.044a3.456 3.456 0 0 1 .457.248l.009.006a3.766 3.766 0 0 1 .754.636l.112.129.093.117.106.145.004.006c.134.196.249.405.343.622.01.028.024.055.035.083l.032.084.002.004.026.072.026.076A3.605 3.605 0 0 1 24 6.69v10.618c0 2.035-1.69 3.69-3.77 3.69h-1.6v-3.614h-7.64V21h-1.6c-2.08 0-3.77-1.656-3.77-3.691v-3.34h12.114V7.584h-1.59v.874h-.84v-.874H13.96v1.485Zm-9.956 1.807c-.662-.688-1.578-.533-2.209 0-2.105 2.012 1.338 5.44 3.302 1.418-.158-.329-.594-.975-1.093-1.418Z"/></svg>
```

Comment:
There are a lot of extraneous nodes that might be worth removing. Ex.
![image](https://github.com/simple-icons/simple-icons/assets/12817388/de8b4dca-0170-4173-b851-1b45ae7bbfac)

Reply:
Removed extraneous nodes